walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,66 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
```

Comment:
Is there a specific reason you cannot pass in List<> instead?
I feel you can get rid of a lot of new ArrayList<> wrapping in this file.

Prefer to choose a sub-interface List, Set, instead of using Collection.

Reply:
This was the original method signature provided, and I wasn't sure if we should change it